src/components/ebay-page-notice/README.md

```diff
@@ -34,8 +34,10 @@ The `<ebay-page-notice>` is a tag used to create a custom-designed notice elemen
 
 ## ebay-page-notice Attributes
 
-| Name             | Type   | Stateful | Required | Description                                                                                    |
-| ---------------- | ------ | -------- | -------- | ---------------------------------------------------------------------------------------------- |
-| `status`         | String | No       | No       | "attention" (default for "page" and "inline"), "confirmation" "information", or "celebration". |
-| `a11y-icon-text` | String | No       | Yes      | adding description for the notice for a11y users                                               |
-| `icon`           | String | Yes      | No       | "default" (matches whatever is specified by the "status") or "none"                            |
+| Name              | Type   | Stateful | Required | Description                                                                                    |
+| ----------------- | ------ | -------- | -------- | ---------------------------------------------------------------------------------------------- |
+| `status`          | String | No       | No       | "attention" (default for "page" and "inline"), "confirmation" "information", or "celebration". |
+| `a11y-icon-text`  | String | No       | Yes      | adding description for the notice for a11y users                                               |
+| `icon`            | String | Yes      | No       | "default" (matches whatever is specified by the "status") or "none"                            |
+| `cta`             | String | No       | No       | adding a dedicated CTA link in notice with full anchor tag in HTML                             |
+| `a11yDismissText` | String | No       | No       | adding a dismiss button to dismiss the notice                                                  |
```

Comment:
Lets remove the page-notice-attributes. We should only have documentation on storybook

Look at `split-button` docs, lets copy the readme from there. Have it have links to point to storybook.

Reply:
That sounds much better - single source. Will do. What's odd is that I don't recall writing the docs for the first few. I think I just added the last 2. Weird.

Reply:
Yes. 
And don't worry about it. I have started doing that method in a couple of components but not for all of them. So as we touch code, we should update them to use the new method (like I did in my dialog PR)
I believe the docs for all the others are just format/whitespace changes. Our prettier changes the tables to align correctly.